bokeh/properties.py

```diff
@@ -544,13 +553,13 @@ def reset_changed_vars(self):
         self._changed_vars = set()
 
     def properties_with_values(self):
-        return dict([ (attr, getattr(self, attr)) for attr in self.properties() ])
+        return dict([ (attr, getattr(self, attr)) for attr in self.properties() if self.lookup(attr).serialized ])
```

Comment:
This changes the semantics of `properties_with_values()`, because now it's serializable properties with values. I presume this function isn't used in other contexts where the original meaning matters (say `HasProps._clone()`)?